server/cache_utils.py

```
@@ -0,0 +1,1093 @@
+        print("[WARNING] Enhanced runner not available, falling back to subprocess mode")
+        print(f"Running model: {model_name}")
+        if commit_hash:
+            print(f"Hash: {commit_hash}")
+        print(f"Cache path: {model_path}")
+
+        if interactive or prompt is None:
+            print("Interactive mode not supported in fallback mode")
+            prompt = prompt or "Hello"
+
+        print(f"Prompt: {prompt}\n")
+        os.system(f'python -m mlx_lm generate --model "{model_path}" --prompt "{prompt}"')
```

Ruff (0.14.1)

Starting a process with a shell, possible injection detected

(S605)